src/main.zig

```diff
@@ -612,6 +612,7 @@ fn buildOutputType(
     var subsystem: ?std.Target.SubSystem = null;
     var major_subsystem_version: ?u32 = null;
     var minor_subsystem_version: ?u32 = null;
+    var treat_as_file_ext: Compilation.FileExt = .unknown;
```

Comment:
This is an optional switch that overrides the autodetection mechanism, using an optional type is better than initializing this variable in a known-invalid state.

```suggestion
    var force_language: ?Compilation.FileExt = null;
```